experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -96,12 +96,28 @@ export interface HasLineage {
 
 export const HasLineage = Type.Object({ lineage: Type.Optional(Type.Array(LineageEvent)) });
 
+export interface IdRange {
+	id: ChangesetLocalId;
+	count: number;
+}
+
+export const IdRange = Type.Object({
+	id: ChangesetLocalIdSchema,
+	count: Type.Number(),
+});
+
 /**
  * @alpha
  */
-export interface CellId extends ChangeAtomId, HasLineage {}
+export interface CellId extends ChangeAtomId, HasLineage {
+	adjacentCells?: IdRange[];
```

Comment:
Consider adding some doc comment to mention the fact that the `CellId`s that make up the range all share the same `IdRange[]` instance.

Reply:
That's an implementation detail which we are not currently relying on anywhere. Why does it need to be documented?

Reply:
I don't think it really *needs* to be, but it's a largely invisible fact that (a) ensures we don't get N² data when someone makes a transaction that deletes a bunch of contiguous nodes, and (b) could lead to subtle bugs if someone were to mutate one without realizing they're mutating all of them (which admittedly seems like an unlikely situation).